### PR TITLE
[FEATURE]: newbee prop

### LINKED ISSUE #567: [FEATURE]: newbee prop
## Summary

The team admin can select the members that are newbee's, when sweeping the newbee button. This prop must be saved on the user that will be added/edited to the team.

### Motivation

A newbee member will not be selected as responsible for the SPLIT sub-teams.

### PR DESCRIPTION
**Pull request by CatiaAntunes96**
Relates to #567

## Proposed Changes

  - The team admin can select the members that are newbee's, when sweeping the newbee button.
  - Refactor of the code of 'ListMembers' component
 

This pull request closes #567

### CODE REVIEW

**Review by nunocaseiro — APPROVED** [review]
LGTM 🚀

File: frontend/src/pages/teams/new.tsx [hunk 1]
```diff
@@ -33,20 +33,24 @@ const NewTeam: NextPage = () => {
...
-					user,
-					role: TeamUserRoles.ADMIN
-				});
-			}
-			return { ...user, isChecked: user._id === session?.user.id };
-		});
+		const listMembers: TeamUser[] = [];
+
+		if (data) {
+			const usersWithChecked = data?.map((user) => {
```
Comment (mourabraz) [inline frontend/src/pages/teams/new.tsx:39]: Is it doing one thing or two things? It seems to me that this "map" I not only transforming the data but is using the loop to do something else. Hum...

**Review by mourabraz — APPROVED** [review]
LGTM 🚀

**Review by nunocaseiro — APPROVED** [review]
LGTM 🚀

**Pull request merged**